docs/content/preview/migrate/migrate-steps.md

```diff
@@ -224,6 +224,30 @@ Refer to [export data](../reference/yb-voyager-cli/#export-data) for details abo
 
 The options passed to the command are similar to the [`yb-voyager export schema`](#export-schema) command. To export only a subset of the tables, pass a comma-separated list of table names in the `--table-list` argument.
 
+#### Accelerate data export for MySQL and Oracle
+
+For MySQL and Oracle, you can optionally speed up data export by setting the environment variable `EXPERIMENTAL_FAST_EXPORT=1` when you run `export data` using yb-voyager.
```

Comment:
@aishwarya24 can you please replace all instances of `EXPERIMENTAL_FAST_EXPORT` to `BETA_FAST_DATA_EXPORT` ? We discussed internally decided that this was more inviting for users to try out.